pkgs/os-specific/linux/apparmor/default.nix

```diff
@@ -2,15 +2,18 @@
 , pkgconfig, which
 , flex, bison
 , linuxHeaders ? stdenv.cc.libc.linuxHeaders
-, python
 , gawk
-, perl
+, withPerl ? stdenv.hostPlatform == stdenv.buildPlatform, perl
+, withPython ? stdenv.hostPlatform == stdenv.buildPlatform, python
 , swig
 , ncurses
 , pam
 , buildPackages
 }:
 
+assert withPerl -> perl.meta.availible or false;
+assert withPython -> python.meta.availible or false;
```

Comment:
*available

Reply:
Isn't this redundant? It will throw an error anyway if not.